deploy-manage/security/set-up-basic-security.md

```diff
@@ -17,26 +17,45 @@ Configuring TLS between nodes is the basic security setup to prevent unauthorize
-* [Generate a Certificate Authority (CA) and a server certificate using the `elasticsearch-certutil` tool](#generate-certificates).
+* [Either provide or generate security certificates](#obtain-certificates).
 * [Configure your {{es}} nodes to use the generated certificate for the transport layer](#encrypt-internode-communication).
 
 Refer to [Transport TLS/SSL settings](elasticsearch://reference/elasticsearch/configuration-reference/security-settings.md#transport-tls-ssl-settings) for the complete list of available settings in {{es}}.
 
-## Generate the certificate authority [generate-certificates]
+::::{note}
+For other deployment types, such as {{ech}}, {{ece}}, or {{eck}}, refer to [](./secure-cluster-communications.md).
+::::
+
+## Obtain certificates
 
 You can add as many nodes as you want in a cluster but they must be able to communicate with each other. The communication between nodes in a cluster is handled by the transport module. To secure your cluster, you must ensure that internode communications are encrypted and verified, which is achieved with mutual TLS.
 
 In a secured cluster, {{es}} nodes use certificates to identify themselves when communicating with other nodes.
 
 The cluster must validate the authenticity of these certificates. The recommended approach is to trust a specific certificate authority (CA). When nodes are added to your cluster they must use a certificate signed by the same CA.
 
-For the transport layer, we recommend using a separate, dedicated CA instead of an existing, possibly shared CA so that node membership is tightly controlled. Use the `elasticsearch-certutil` tool to generate a CA for your cluster.
+For the transport layer, we recommend using a separate, dedicated CA instead of an existing, possibly shared CA so that node membership is tightly controlled.
+
+When you manually set up transport TLS, you can choose from the following CA options: 
+
+* [Provide certificates from a private or third-party CA](#private-3p)
+* [Use the `elasticsearch-certutil` tool to generate a CA unique to your cluster](#generate-certificates)
+
+### Provide certificates from a private or third-party CA [private-3p]
+
+You might choose to use a private or third-party CA to generate transport certificates for node-to-node connections.
+
+Transport connections between {{es}} nodes are security-critical and you must protect them carefully. Malicious actors who can observe or interfere with unencrypted node-to-node transport traffic can read or modify cluster data. A malicious actor who can establish a transport connection might be able to invoke system-internal APIs, including APIs that read or modify cluster data.
+
+Carefully review [](/deploy-manage/security/self-tls-considerations.md) to ensure that the certificates that you provide meet the security requirements for transport connections.
+
+After you obtain your certificate, place the certificate file in the `$ES_PATH_CONF` directory on **every** node in your cluster. 
```

Comment:
... and configure Elasticsearch to use these files. Also it's not just the certificate you obtained, you also need to tell Elasticsearch which CA certificates it can trust, and it needs the private key too. TBH I think it'd be simpler to drop this sentence, it is adequately covered elsewhere.